vitals/Main.java

```diff
@@ -1,23 +1,36 @@
 package vitals;
 
 public class Main {
```

Comment:
batteryIsOk() is a convenience method that abstracts different parameter status check from the user. The use could just call an api and know the health of the battery. But in this case, since we do not have a single method, user is forced to use all 3 parameter APIs to check status and then decide based on the result of those three outcomes.
Please think over the consequences of eliminating batteryIsOk() API.